microscopy_SEM001/sub-01/sub-01_sessions.tsv

```diff
@@ -0,0 +1,2 @@
+session_id	acq_time
```

Comment:
It is a bit weird to have only one session folder. Generally (not always), the session folder is used when there are more than one. Maybe we would had a `ses-02` with the same subject/sample.